lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -2061,17 +2082,22 @@ bool DWARFASTParserClang::ParseTemplateDIE(
         std::optional<uint64_t> size = clang_type.GetBitSize(nullptr);
         if (!size)
           return false;
```

Comment:
It looks like this exists only to be passed to `MakeAPValue`. Since it already gets `clang_type` as an argument (and uses it to retrieve things like bit width and float semantics), could we move this there as well?